github-actions/unified-status-check/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Unified Status Check'
+description: 'Monitor the statuses on a pull request to ensure all expected statuses are passing'
+author: 'Angular'
+inputs:
+  angular-robot-key:
+    description: 'The private key for the Angular Robot Github app.'
+    required: true
+  ignored:
+    description: 'Status matchers to be ignored.'
```

Comment:
Nit: Can you clarify in the description what a matcher is. e.g. regex pattern or what it takes. same below